benchmarks/src/find_first_of.cpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <algorithm>
+#include <benchmark/benchmark.h>
+#include <cstddef>
+#include <cstdint>
+#include <numeric>
+
+using namespace std;
+
+template <class T, size_t Pos, size_t NSize, size_t HSize = Pos * 2, size_t Which = 0>
+void bm(benchmark::State& state) {
+    vector<T> h(HSize, T{'.'});
+    vector<T> n(NSize);
+    iota(n.begin(), n.end(), T{'a'});
+
+    static_assert(Pos < HSize);
+    static_assert(Which < NSize);
+    h.at(Pos) = n.at(Which);

```

Comment:
After the `static_assert`s, `at()` is super unnecessary, it should just be square brackets.